templates/pages/example.html.twig

```diff
@@ -32,6 +32,7 @@
     <p>{{ content.color }}</p>
     <p>{{ content.time }}</p>
     <p>{{ content.date }}</p>
+    <p>{{ content.datetime }}</p>
```

Comment:
As this was changed to datetime (not sure if a good idea). This twig file need to be change to the following:

```twig
{{ content.datetime  ? content.datetime.format('Y-m-d H:i') : '' }}
```

Reply:
How else would you allow to properly format the string otherwise? The content type itself cannot know if you want to write e.g. the month in a numeric or spelled out way. And this way also [twig functionality can be reused](https://twig.symfony.com/doc/2.x/filters/format_datetime.html).

Reply:
adjusted 🙂